tests/test_objective_funs.py

```diff
@@ -1602,422 +1527,344 @@ def test_loss_function_asserts():
         RotationalTransform(eq=eq, loss_function=fun)
 
 
-@pytest.mark.unit
-@pytest.mark.slow
-def test_compute_scalar_resolution():  # noqa: C901
```

Comment:
no functionality change here just broke it up into smaller pieces for parallelization and reducing code duplication

Reply:
this will also make sure that any new added objectives get tested as well